### PR TITLE
Make "load into s3" great again

### PR DESCRIPTION
**Pull request by shepwalker**

### CODE REVIEW

File: corji/data_sources/s3.py [hunk 1]
```diff
@@ -68,12 +68,11 @@ def put(emoji, corgis, override_existing_file=False):
 
     Returns True if at least one put is successful; returns False otherwise.
     """
-
+
```
Comment (jmduke) [inline corji/data_sources/s3.py]: remove these spaces on the empty lines

File: corji/data_sources/s3.py [hunk 2]
```diff
@@ -112,7 +111,7 @@ def put(emoji, corgis, override_existing_file=False):
             else:
                 logger.debug("%s found in remote cache. Skipping", s3_key)
 
-        except (HTTPError, ConnectionError, requests.exceptions.ConnectionError) as e:
+        except (HTTPError, ConnectionError, requests.exceptions.ConnectionError, ValueError) as e:
```
Comment (jmduke) [inline corji/data_sources/s3.py]: can we swap these exceptions and just catch OSError, then catch errything?

File: corji/data_sources/s3.py [hunk 3]
```diff
@@ -160,4 +160,4 @@ def get(emoji):
 
 
 def get_file_name_from_emoji(i, raw_emoji):
-    return emoji.demojize(raw_emoji).replace(":", "") + "/0{}.jpg".format(i + 1)
+    return raw_emoji + "/{:0>2d}.jpg".format(i + 1)
```
Comment (jmduke) [inline corji/data_sources/s3.py]: comment explaining what dis do

File: corji/data_sources/s3.py [hunk 4]
```diff
@@ -121,7 +120,8 @@ def put(emoji, corgis, override_existing_file=False):
 
 
 def get_all(raw_emoji):
-    folder_name = emoji.demojize(raw_emoji).replace(":", "")
+    folder_name = get_file_name_from_emoji(1, raw_emoji).split('/')[0]
```
Comment (jmduke) [inline corji/data_sources/s3.py]: comment explaining desired format of string

File: corji/scripts/gather_usage_stats.py [hunk 5]
```diff
@@ -23,4 +23,4 @@
     with open(TARGET_FILE, "w+", newline = '') as csvfile:
         csvwriter = csv.writer(csvfile)
         for message in client.messages.iter(page_size = PAGE_SIZE): 
-            csvwriter.writerow([message.from_, message.to, message.body, message.date_sent, message.num_media, message.status, message.direction, message.error_message or ""])
\ No newline at end of file
+            csvwriter.writerow([message.sid, message.from_, message.to, message.body, message.date_sent, message.num_media, message.status, message.direction, message.error_message or ""])
```
Comment (jmduke) [inline corji/scripts/gather_usage_stats.py]: did you mean to include this?

File: corji/scripts/load_into_s3.py [hunk 6]
```diff
@@ -19,21 +23,36 @@
...
+            s3.put(item, corgis[item])
+        except Exception as e:
+            logger.error("Error!", e)
+        finally:
+            queue.task_done()
+
+def create_bucket_if_not_exist():
+    aws_s3_client = boto3.client("s3")
+    aws_s3_resource = boto3.resource("s3")
+    relevantBuckets= [bucket for bucket in aws_s3_resource.buckets.all() if(bucket.name==Config.AWS_S3_CACHE_BUCKET_NAME)]
```
Comment (jmduke) [inline corji/scripts/load_into_s3.py]: fix spacing, no camelcase

File: corji/scripts/gather_usage_stats.py (hunk 5 above)
Comment (shepwalker) [inline corji/scripts/gather_usage_stats.py]: No but I don't think that's a massive problem?

File: corji/data_sources/s3.py (hunk 1 above)
Comment (shepwalker) [inline corji/data_sources/s3.py]: done

**Pull request merged**